src/app/components/BalanceBox/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAccount } from "~/app/context/AccountContext";
+
+function BalanceBox() {
+  const { balancesDecorated } = useAccount();
+
+  const [balance, sats] = balancesDecorated.accountBalance.split(" ");
```

Comment:
I think you could use `account.balance` from `useAccount()` for that instead of manually splitting those values which seems a bit prone to errors.

Reply:
Hello @reneaaron,

I tried using ``account.balance`` from ``useAccount()``, but it only returned the balance amount without the ``sats`` string after it and without proper comma formatting in the balance. Please take a look at the screenshot below:

![new](https://user-images.githubusercontent.com/81313264/230733121-1c1c8f5d-5bb0-4e9d-a990-b3cdfc107388.png)


Code:

`````import { useAccount } from "~/app/context/AccountContext";
   function BalanceBox() {
  const { balancesDecorated, account } = useAccount();
  const hasFiatBalance = Boolean(balancesDecorated.fiatBalance);
  const balance = account?.balance;
  

 return (
    <div
      className={`w-full flex flex-col items-center justify-center bg-white dark:bg-surface-02dp dark:text-white shadow rounded-md  ${
        hasFiatBalance ? "" : "min-h-[88px]"
      }`}
    >
      <div className={hasFiatBalance ? "mt-4 mb-2" : "mt-0 mb-0"}>
        <span className="text-xl font-medium ">
          {balance}
        </span>
      </div>
      {balancesDecorated.fiatBalance && (
        <span className="text-sm font-normal text-gray-500 mb-4 ">
          ~{balancesDecorated.fiatBalance}
        </span>
      )}
    </div>
  );
}

export default BalanceBox;````

Reply:
Hello@reneaaron ,
can you help with this ?